testing/src/main/java/net/consensys/linea/testing/SolidityUtils.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Consensys Software Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package net.consensys.linea.testing;
+
+import java.net.URL;
+import java.util.Iterator;
+import java.util.Map;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.tuweni.bytes.Bytes;
+import org.hyperledger.besu.evm.log.LogTopic;
+import org.web3j.protocol.core.methods.response.Log;
+import org.web3j.tx.Contract;
+
+public class SolidityUtils {
+  private static final ObjectMapper objectMapper = new ObjectMapper();
+  private static final ClassLoader classLoader = SolidityUtils.class.getClassLoader();
+
+  public static Bytes getContractByteCode(Class<? extends Contract> contractClass) {
```

Comment:
I'm not super clear on what's happening here.  I think it would be helpful to write some amount of documentation to explain (at a high level) what's going on.

Reply:
I agree with this.

Reply:
Added some documentation for this method.